externs/air/sys/FileSystem.hx

```diff
@@ -37,5 +37,5 @@ class FileSystem
 
 	public static function deleteDirectory(path:String):Void {}
 
-	public static function readDirectory(path:String):Array<String> {}
+	public static function readDirectory(path:String):Array<String> {return [];}
```

Comment:
Not necessary. If this is causing errors, something is set up incorrectly on your end.